portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Blog - Anusha Datta</title>
+
+    <!-- Favicon, CSS & JS -->
+    <link rel="icon" href="images/AD.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/blog.css">
+    <script src="script.js"></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
+</head>
+
+<body>
+
+    <!-- Heading -->
+    <header id="home" class="showcase">
+        <div class="content">
+
+            <img src="images/AD.png" class="logo" alt="AD">
+            <h1 class="title">
+                BLOG
+            </h1>
+
+        </div>
+    </header>
+
+    <!-- List of Posts -->
+    <section class="bg-light">
+        <div class="container center">
+            <h1>COMING SOON</h1>
+            <p>Stay tuned for some exciting content!</p>
+        </div>
+
+        <hr class="center">
+    </section>
+
+    <!-- Contact -->
+    <section class="bg-light" id="contact">
+        <div class="container center">
+
+            <a href="mailto:anushadatta@gmail.com"><i class="fa fa-envelope"></i></a>
+
+            <a href="https://www.linkedin.com/in/anusha-datta/" target="_blank"><i class="fa fa-linkedin"></i></a>
+
+            <a href="https://github.com/anushadatta" target="_blank"><i class="fa fa-github"></i></a>
+
+            <a href="https://medium.com/@anushadatta" target="_blank"><i class="fa fa-medium"></i></a>
+
+        </div>
+    </section>
+
+    <!-- Footer -->
+    <footer class="center bg-dark">
```

Comment:
Optional, If you have several web pages with common header footer or other sections, consider declaring them at one place and re-use.
Remember the principle DRY-Don't repeat yourself.
Although I don't blieve we can do this using pure html, we may need to use server side language like PHP or some framework for this kind of templating. Hence this comment is optional.

Reply:
Yes, I'm not sure I can enforce that here, but I'll definitely keep that in mind for any future projects I do. Thank you for that notable insight!